crates/sui-config/src/builder.rs

```diff
@@ -97,6 +97,7 @@ impl<R: ::rand::RngCore + ::rand::CryptoRng> ConfigBuilder<R> {
                 ValidatorInfo {
                     public_key,
                     stake,
+                    delegation: 0, // no delegation yet at genesis
```

Comment:
Do we need two separate items though? What if stake always include both self stake and delegation?